test/helpers/assertRevert.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  assertRevert: async (promise) => {
+    try {
+      await promise;
+      assert.fail('Expected revert not received');
```

Comment:
This should go after the catch block, and the inside of the catch block should have a return; at the end. Otherwise this error, which contains the substring revert, will pass the assertion on line 8 (which it should not, we want the test to fail if we ever hit line 5).